tensorboard/plugins/hparams/hparams_plugin.py

```diff
@@ -116,14 +117,24 @@ def get_experiment_route(self, request):
             request_proto = _parse_request_argument(
                 request, api_pb2.GetExperimentRequest
             )
+            response_proto = get_experiment.Handler(
+                ctx,
+                self._context,
+                experiment_id,
+                request_proto,
+            ).run()
+            current_protobuf_version = importlib_version("protobuf")
```

Comment:
Consider adding a util that we can use as a one-line call in all of these places, and abstract this detail away.

It's annoying that we'd need a separate file and all the overhead of the build file and all, but I think it's a more maintainable solution and results in cleaner code.

Reply:
There's already a `plugin_util` module, so I added it there (and added some tests).

Reply:
Sorry for not being clear. I was thinking more like a "proto_to_json" method, so the package checking is abstracted away from the code that just needs to convert the proto to a string, and the code elsewhere can be just a one-line method call. Sorry for the churn.

Reply:
I've updated to just have a `proto_to_json` method, but I've only used it for these two specific instances that use a deprecated field conditionally. There are probably other calls to `MessageToJson` throughout the codebase that don't need the conditional handling, but avoiding touching those as part of this PR.